packages/compartment-mapper/test/test-bundle.js

```diff
@@ -77,3 +77,44 @@ test('equivalent archive behaves the same as bundle', async t => {
   });
   t.deepEqual(log, expectedLog);
 });
+
+// This is failing because it requires support for missing dependencies.
+// Cannot bundle: encountered deferredError Cannot find file for internal module "./spam"
+test.failing('bundle cjs-compat', async t => {
+  const cjsFixture = new URL(
+    'fixtures-cjs-compat/node_modules/app/index.js',
+    import.meta.url,
+  ).toString();
+
+  const bundle = await makeBundle(read, cjsFixture);
+  const log = [];
+  const print = entry => {
+    log.push(entry);
+  };
+  const compartment = new Compartment({ print });
+  compartment.evaluate(bundle);
+  t.deepEqual(log, expectedLog);
+});
+
+test('bundle cjs-compat default-difficulties', async t => {
+  const cjsFixture = new URL(
+    'fixtures-cjs-compat/node_modules/default-difficulties/index.mjs',
+    import.meta.url,
+  ).toString();
+
+  const bundle = await makeBundle(read, cjsFixture);
+  const compartment = new Compartment();
+  const { results } = compartment.evaluate(bundle);
+  const resultExports = results.map(result => {
+    return Object.keys(result).sort();
+  });
+  t.deepEqual(resultExports, [
+    ['default', 'even'],
```

Comment:
if required in cjs they should not expose the default unless it was explicitly assigned by the module. I think that's true with the current implementation. Not sure if there's a fixture for that, but there might already be.

Reply:
im surprised `default` is not like a Symbol or something, but I dont know the desired behavior for compat